src/data.cljs

```diff
@@ -527,16 +527,20 @@
 "Kap@⍥ (over)@-@5740" {:lang "Kap" :algo "⍥ (over)" :lib "-" :id 5740 :doc "https://kapdemo.dhsdevelopments.com/reference.html#_over" :sig "-" :expr false}
 "J@&: (appose)@-@5740" {:lang "J" :algo "&: (appose)" :lib "-" :id 5740 :doc "https://code.jsoftware.com/wiki/Vocabulary/ampco" :sig "-" :expr false}
 "BQN@○ (over)@-@5740" {:lang "BQN" :algo "○ (over)" :lib "-" :id 5740 :doc "https://mlochbaum.github.io/BQN/doc/compose.html" :sig "-" :expr false}
+"Uiua@∩ (both)@-@5740" {:lang "Uiua" :algo "∩ (both)" :lib "-" :id 5740 :doc "https://www.uiua.org/docs/both" :sig "-" :expr false}
```

Comment:
You don't actually need to run `gen.py` locally. When a change is committed to a GH Action runs the script and then commits an updated `data.cljs`. It is also fine to update it manually if you want but it might lead to more conflicts (hence why I am mentioning it). So probably easier to just revert changes to this file (or resolve conflict).

Reply:
ah alright